pkgs/by-name/ty/typespec/package.nix

```
@@ -0,0 +1,100 @@
+{
+  fetchFromGitHub,
+  lib,
+  makeWrapper,
+  nodejs,
+  pnpm,
+  stdenvNoCC,
+  testers,
+  typespec,
```

`lib` is customarily the first argument, and `stdenv(NoCC)` second
```suggestion
  lib,
  stdenvNoCC,
  fetchFromGitHub,
  makeWrapper,
  nodejs,
  pnpm,
  testers,
```